### PR TITLE
-  Modification of events.component.html

### PR DESCRIPTION
**Pull request by DentVega**
-  Modification of events.component.scss
-  Modification of events.model.ts
-  Modification of en.json
-  Modification of es.json

### CODE REVIEW

**Comment by DentVega** [comment]
@mauricioarce solicito una revision por favor

File: src/app/home/events/events.component.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@import '../../../styles-mat-variables';
```
Comment (mauricioarce) [inline src/app/home/events/events.component.scss]: If you are not using this import, please get rid of it.

File: src/app/home/events/events.component.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { Component, OnInit } from '@angular/core';
 
 import { Observable } from 'rxjs';
 
+import {LanguageService} from '../../core/services/language.service';
```
Comment (mauricioarce) [inline src/app/home/events/events.component.ts]:
```suggestion
import { LanguageService } from '../../core/services/language.service';
```

**Comment by DentVega** [comment]
@mauricioarce correcciones realizadas

**Pull request merged**